metricbeat/_meta/beat.full.yml

```diff
@@ -94,6 +94,27 @@ metricbeat.modules:
   #period: 10s
   #hosts: ["localhost:9092"]
 
+  #client_id: metricbeat
+
+  #metadata.retries: 3
```

Comment:
Not sure if it we should use metadata namespace here, as in the end from a user perspective it is just retrieds. Metadata is like an internal "logic" thing.